test/e2e/specs/block-icons.test.js

```diff
@@ -130,7 +130,7 @@ describe( 'Correctly Renders Block Icons on Inserter and Inspector', () => {
 		it( 'Renders the icon in the inserter with the correct background color and an automatically compute readable foreground color', async () => {
 			await searchForBlock( blockTitle );
 			validateSvgIcon( await getFirstInserterIcon() );
-			expect( await getBackgroundColor( INSERTER_ICON_SELECTOR ) ).toEqual( 'rgb(1, 0, 0)' );
+			expect( await getBackgroundColor( INSERTER_ICON_SELECTOR ) ).toEqual( 'rgba(0, 0, 0, 0)' );
```

Comment:
The background color is in fact rgb(1, 0, 0) and with this change we don't verify if the background was in fact applied. I think we may need to update the selector const INSERTER_ICON_SELECTOR = `${ INSERTER_BUTTON_SELECTOR } .editor-block-icon`; to be  `${ INSERTER_BUTTON_SELECTOR } .editor-block-types-list__item-icon`.

Reply:
Oh gotcha I'll take a look

Reply:
.editor-block-icon is a child of .editor-block-types-list__item-icon and it seems colors are applied in .editor-block-types-list__item-icon.

Reply:
It should be fixed.